src/applications/deploying-apps.md

```diff
@@ -80,6 +77,10 @@ Kubernetes takes care of the entire deploy, doing rolling updates in the backgro
 
 The base timeout is multiplied with `DEIS_DEPLOY_BATCHES` to create an overall timeout. This would be 240 (timeout) * 4 (batches) = 960 second overall timeout.
 
+#### RC deploy
+
+This deploy timeout defines how long to wait for each batch to complete in `DEIS_DEPLOY_BATCHES`
```

Comment:
Could you add a period at the end of this sentence? Nitpick alert! Otherwise LGTM.